### PR TITLE
mention google translate

### PR DESCRIPTION
**Pull request by lsmith77**
also remove mention of unofficial translations since these are no longer accessible

to further simplify the use of google translate, we might want to consider to add a drop down with the lets say top 5 languages spoken in the countries that most frequently visit symfony.com .. or we might even provide a dynamic "translate" button by looking at the browser language settings.

### CODE REVIEW

**Review by weaverryan — APPROVED** [review]
I think it’s nice - clarifies our position

**Review by fabpot — APPROVED** [review]

File: contributing/documentation/translations.rst [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 ============
 
-The official Symfony documentation is published only in English, but some
-community groups maintain `unofficial translations`_.
+The official Symfony documentation is published only in English. You can
+read about the reasons in `this blog post`_.
 
-.. _`unofficial translations`: https://symfony.com/blog/discontinuing-the-symfony-community-translations
+We have taken steps to improve the experience when using
+`google translate`_ to prevent code blocks from being translated.
```
Comment (javiereguiluz) [inline contributing/documentation/translations.rst]: `google translate` -> `Google Translate`  ... and there's other occurrence below.

**Comment by javiereguiluz** [comment]
@lsmith77 about this idea:

> to further simplify the use of google translate, we might want to consider to add a drop down with the lets say top 5 languages spoken in the countries that most frequently visit symfony.com .. or we might even provide a dynamic "translate" button by looking at the browser language settings.

It'd be great to be 100% inclusive about language, but the truth is that having a basic English level is one of the requisites to use Symfony. Having a medium PHP level is another requisite.

To make things easier for people, we're super careful to not use idioms or uncommon English words in the docs (I've asked @weaverryan several times to simplify some of the words he used in some of his contributions).

If someone can't read the Symfony Docs in English, they won't be able to use Symfony: command names are in English, command help is in English, error pages are in English, the profiler and debug toolbar are in English, all foundational concepts are in English and lots of them are hard to translate to other languages: "event dispatcher", "service decoration", "message bus", "routing", etc.

**Comment by lsmith77** [comment]
I am not an expert and I will try to seek some articles on this topic, but I would assume its not a “binary” thing, ie. people may need to speak a bit of english but sometimes they will struggle and then having a translation helps both to understand something but also to improve their english.

as for jargo terms like “event dispatcher” .. I think those will also be learned.

furthermore if we can bing down the barrier to entry we can also create motivation to learn and improve their english skills. I remember especially on IRC we had a lot of people that steadily improved their language skills over months of interacting via chat.

finally such efforts are also a way that we can show that we care about people that speak other languages natively. in a way its a token of respect.

**Comment by lsmith77** [comment]
so my tl;dr on this topic: lets try to do as much as we can reasonably can. so f.e. if someone implements such a translation button I don’t think it would be a burden to have it. and we could also track how often it is used to see how useful it is.

File: contributing/documentation/translations.rst (hunk 1 above)
Comment (lsmith77) [reply contributing/documentation/translations.rst]: fixed

**Comment by jwage** [comment]
I thought this was a good idea so I implemented it on www.doctrine-project.org. Just thought I would share if you all wanted to do something similar.

**Comment by lsmith77** [comment]
@jwage would be interesting to collect some analytics data on how often this feature is used and what languages are picked most often.

**Comment by jwage** [comment]
I will add some Google analytics tracking to it and report back with the results.

**Comment by javiereguiluz** [comment]
Thanks Lukas.

**Comment by lsmith77** [comment]
@jwage I guess it does show like its being used .. though the fact that some people translate to english seems odd.

**Pull request merged**